flocker/node/agents/blockdevice.py

```diff
@@ -1121,8 +1121,10 @@ def list_live_nodes():
         This is used to figure out which nodes are dead, so that other
```

Comment:
:grey_question: Potentially awkward interface note. This is kind of like a `Get details on the nodes, and filter results so that I only see the live nodes`. For instance, if this was directly an interface on GCE I would probably be a bit upset.

I think more natural would be 'list_nodes' with an optional filter for only live nodes.